From ee0b38073853f655f201f9f30324b1d552f707f1 Mon Sep 17 00:00:00 2001
From: Josef Schlehofer <pepe.schlehofer@gmail.com>
Date: Wed, 21 Jul 2021 22:43:09 +0200
Subject: [PATCH 2/4] foris-client: bump to version 1.0.4

1.0.4 (2021-07-21)
------------------

* move ubus socket path due to upstream change
* change URL from gitlab.labs.nic.cz to gitlab.nic.cz
---
 web/foris-controller/foris-client/Makefile | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/web/foris-controller/foris-client/Makefile b/web/foris-controller/foris-client/Makefile
index 93bccfa89..91cfe1245 100644
--- a/web/foris-controller/foris-client/Makefile
+++ b/web/foris-controller/foris-client/Makefile
@@ -8,12 +8,12 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=foris-client
-PKG_VERSION:=1.0.3
-PKG_RELEASE:=4
+PKG_VERSION:=1.0.4
+PKG_RELEASE:=1
 
 PKG_SOURCE_PROTO:=git
 PKG_SOURCE_URL:=https://gitlab.nic.cz/turris/foris-controller/foris-client.git
-PKG_MIRROR_HASH:=715f5c40d7950fb781ea5df477396ade63deb99812ccc80690278eb337c8ada3
+PKG_MIRROR_HASH:=761775de882c6ef66d5a05ce81a04ebbcd99f7114d9d39b6d8b2375542e7740c
 PKG_SOURCE_VERSION:=v$(PKG_VERSION)
 
 PKG_MAINTAINER:=CZ.NIC <packaging@turris.cz>
-- 
2.30.2

